misc/well-known

```diff
@@ -0,0 +1,3 @@
+{
+  "m.server": "matrix.ocf.berkeley.edu:443"
+}
```

Comment:
what is this?

Reply:
Currently unused file that could be used for federation, just kept this here for reference